### PR TITLE
Add JIRA setting to show dashboard instead of create issue

### PR DESCRIPTION
**Pull request by zackhsi**
My most common JIRA action is to view my dashboard, so that's what the
command `jira` should do.

### CODE REVIEW

**Comment by zackhsi** [comment]
// @robbyrussell @mcornella

File: plugins/jira/jira.plugin.zsh [hunk 1]
```diff
@@ -38,8 +42,11 @@ open_jira_issue () {
   fi
 
   if [ -z "$1" ]; then
-    echo "Opening new issue"
-    $open_cmd "${jira_url}/secure/CreateIssue!default.jspa"
+    if [[ "x$JIRA_DASHBOARD" = "xtrue" ]]; then
```
Comment (apjanke) [inline plugins/jira/jira.plugin.zsh]: You don't need the "x$FOO" trick in `zsh`. `[[ "$JIRA_DASHBOARD == true ]]` is sufficient.

**Comment by apjanke** [comment]
How about adding a `jira dashboard` special argument (like `jira assigned` and `jira reported`), and a `$JIRA_DEFAULT_ACTION` control instead of `$JIRA_DASHBOARD`? That way people who don't want it as the default action could still use the new feature. And the customization could be used if other special actions get defined, instead of having to reconcile multiple boolean controls. (And people could even use the existing `assigned` and `reported` actions if they wanted.)

**Comment by zackhsi** [comment]
Updated @apjanke

**Pull request closed without merging**